wherehows-web/tsconfig.json

```diff
@@ -29,7 +29,8 @@
       ],
       "wherehows-web/mirage/*": [
         "mirage/*"
-      ]
+      ],
+      "router/*": ["node_modules/router_js/dist/cjs/*"]
```

Comment:
Is this used? Since you are importing from `router_js` and not `router/...`

Reply:
Yes, internally the library needs this...

Reply:
Posted a question in Discord. Will wait to see if there is an easier way.

Reply:
I am reverting this change and manually add the definitions manually.